src/app/movies/movies.component.ts

```diff
@@ -21,10 +23,12 @@ export class MoviesComponent {
   }
 
   parseUserInput(movieInput: string): Movie {
+    const videoMetadata = getVideoId(movieInput);
+    console.log(videoMetadata);
```

Comment:
umawialiśmy się, żebyś wylogowł sobie te dane, ale to tak bardziej żebyś sprawdziłczy działa i jak działa - robiac pull requesty wywalaj wszystkie console.logi (nie zostawia się logów w aplikacjach, to jest niepotrzebne więc warto sie przyzwyczajać)